File: test/items/example/__check__/ts/auth/auth-model.ts
/**
 * User model base
*/
export class UserBase {
  /**
   * Required. 
   */
  UserName: string;
  /**
   * Required. ads
   */
  Psw: boolean;
}
/**
 * User model
*/
export class User extends UserBase {
  /**
   * Required. 
   */
  EMail: string;
}
export interface BaseTypeExt {
  /**
   * desc GetDataBaseExt
   * returns integer result desc
   */
  GetDataBaseExt(): number;
}

File: test/items/example/__check__/cs/auth/auth-model.cs
using System;

namespace Example.Auth
{


  /// <summary>
  /// User model base
  /// </summary>
  public partial class UserBase
  {
    /// <summary>
    /// Required. 
    /// </summary>
    public virtual string UserName {get; set; }
    /// <summary>
    /// Required. ads
    /// </summary>
    public virtual bool Psw {get; set; }
  }
  /// <summary>
  /// User model
  /// </summary>
  public partial class User : UserBase 
  {
    /// <summary>
    /// Required. 
    /// </summary>
    public virtual string EMail {get; set; }
  }


  public partial interface BaseTypeExt
  {
    /// <summary>
    /// desc GetDataBaseExt
    /// <returns>result desc</returns>
    /// </summary>
    int GetDataBaseExt();
  }
}
